<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VESPropertiesPanel</class>
 <widget class="QToolBox" name="VESPropertiesPanel">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ToolBox</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="pageGeneral">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>293</width>
     <height>278</height>
    </rect>
   </property>
   <attribute name="label">
    <string>Propiedades generales</string>
   </attribute>
   <attribute name="toolTip">
    <string>Propiedades generales</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="labelEquipment">
        <property name="text">
         <string>Equipo:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelFieldOperator">
        <property name="text">
         <string>Operador:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelVesDate">
        <property name="text">
         <string>Fecha:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelVesName">
        <property name="text">
         <string>Nombre:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEditEquipment"/>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEditVesName"/>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEditVesOperator"/>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QDateEdit" name="dateEditVesDate">
        <property name="displayFormat">
         <string>dd/MM/yyyy</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="pageCoordinates">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>293</width>
     <height>278</height>
    </rect>
   </property>
   <attribute name="label">
    <string>Ubicación del SEV</string>
   </attribute>
   <attribute name="toolTip">
    <string>Ubicación del SEV</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="labelCoordinateSystem">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Sistema:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="labelLatitude_North">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Latitud:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="3">
       <widget class="QLineEdit" name="lineEditLatNorth"/>
      </item>
      <item row="4" column="4">
       <widget class="QPushButton" name="pushButtonGoogleElevation">
        <property name="text">
         <string>Cota de Google</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="labelLongitude_East">
        <property name="text">
         <string>Longitud:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="3">
       <widget class="QLineEdit" name="lineEditLngEast"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="labelSystemOptions">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Formato:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="3">
       <widget class="QComboBox" name="comboBoxSystemOptions">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="3">
       <widget class="QComboBox" name="comboBoxCoordinateSystem">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="2">
       <widget class="QLineEdit" name="lineEditElevation"/>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="labelElevation">
        <property name="text">
         <string>Cota:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="pageCurrentModel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>293</width>
     <height>278</height>
    </rect>
   </property>
   <attribute name="label">
    <string>Propiedades del modelo seleccionado</string>
   </attribute>
   <attribute name="toolTip">
    <string>Propiedades del modelo seleccionado</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <signal>VESNameEdited(QString)</signal>
 </slots>
</ui>
